src/web/index.js

```diff
@@ -113,9 +125,22 @@ render(
 
 const params = queryString.parse(window.location.search);
```

Comment:
Might drop a long comment here about the startup process and shared theme handling